lib/ontologies_linked_data/serializers/json.rb

```diff
@@ -6,6 +6,21 @@ class JSON
       CONTEXTS = {}
 
       def self.serialize(obj, options = {})
+        
+        result_lang = ''
+
+        if obj.submission
```

Comment:
can you put this block in another place, for example as a method in obj? 

Because it is not related directly to serializing and we want to complexify it more.

Same for the method get_submission_languages